mlrun/feature_store/api.py

```diff
@@ -256,23 +257,29 @@ def get_online_feature_service(
 
             Example with imputing::
 
-                svc = get_online_feature_service(vector_uri, impute_policy={"*": "$mean", "amount": 0))
+                svc = get_online_feature_service(vector_uri, ['id'], impute_policy={"*": "$mean", "amount": 0))
                 try:
                     resp = svc.get([{"id": "C123487"}])
                 except Exception as e:
                     handling exception...
                 finally:
                     svc.close()
 
-    :param feature_vector:    feature vector uri or FeatureVector object. passing feature vector obj requires update
-                            permissions
-    :param run_config:        function and/or run configuration for remote jobs/services
-    :param impute_policy:     a dict with `impute_policy` per feature, the dict key is the feature name and the dict
-                            value indicate which value will be used in case the feature is NaN/empty, the replaced
-                            value can be fixed number for constants or $mean, $max, $min, $std, $count for statistical
-                            values. "*" is used to specify the default for all features, example: `{"*": "$mean"}`
-    :param fixed_window_type: determines how to query the fixed window values which were previously inserted by ingest
-    :param update_stats:      update features statistics from the requested feature sets on the vector. Default: False.
+    :param feature_vector:      feature vector uri or FeatureVector object. passing feature vector obj requires update
+                                permissions.
+    :param run_config:          function and/or run configuration for remote jobs/services
+    :param impute_policy:       a dict with `impute_policy` per feature, the dict key is the feature name and the dict
+                                value indicate which value will be used in case the feature is NaN/empty, the replaced
+                                value can be fixed number for constants or $mean, $max, $min, $std, $count
+                                for statistical
+                                values. "*" is used to specify the default for all features, example: `{"*": "$mean"}`
+    :param fixed_window_type:   determines how to query the fixed window values which were previously inserted by ingest
+    :param update_stats:        update features statistics from the requested feature sets on the vector.
+                                Default: False.
+    :param entity_keys:         Entity list of the first feature_set in the vector.
+                                The indexes that are used in the get function.
```

Comment:
```suggestion
                                The indexes that are used to query the online service.
```

Also, since this is optional, please specify what is used if not provided.